ruth/data/map.py

```diff
@@ -215,7 +215,10 @@ def nearest_nodes(gps):
 
     def shortest_path(self, origin, dest):
         """Compute shortest path between two OSM nodes."""
-        return nx.shortest_path(self.original_network, origin, dest, weight="length")
+        try:
```

Comment:
Doesn't this signal some kind of bug in the input data?

Reply:
It can happen during the generating of the input parquet file.